src/loaders/configure/configure.js

```diff
@@ -28,13 +36,21 @@ export function configure (agentIdentifier, opts = {}, loaderType, forceDrain) {
   const updatedInit = getConfiguration(agentIdentifier)
   runtime.denyList = [
     ...(updatedInit.ajax?.deny_list || []),
-    ...(updatedInit.ajax?.block_internal
-      ? [
-          info.beacon,
-          info.errorBeacon
-        ]
-      : [])
+    ...(updatedInit.ajax?.block_internal ? [info.beacon, info.errorBeacon] : [])
```

Comment:
Does this need to include the proxyBeacon as well, since that would ingest the data if set

Reply:
nvm i see it below